cloud_governance/common/aws/cloudtrail/cloudtrail_operations.py

```diff
@@ -0,0 +1,32 @@
+from datetime import timedelta, datetime
+
+import boto3
+
+
+class CloudTrailOperations:
+    time = 2
```

Comment:
Class constant should be in Uppercase
https://stackoverflow.com/questions/10672419/class-constants-in-python

Please specify with time its: SLEEP_TIME or WAIT_TIME